ward/terminal.py

```diff
@@ -424,6 +447,18 @@ def output_test_result_summary(
 
         print(output)
 
+    def _output_longest_durations(self, test_results: List[TestResult], num_tests: int):
```

Comment:
It'd be nice if we could print this in the same format headers are printed when a test fails ( the "Comparison", "Location" headers etc)

Reply:
Ok, np